fe/fe-core/src/main/java/org/apache/doris/nereids/rules/expression/rules/FunctionBinder.java

```diff
@@ -141,6 +141,9 @@ private UnboundFunction bindHighOrderFunction(UnboundFunction unboundFunction, E
 
     @Override
     public Expression visitUnboundFunction(UnboundFunction unboundFunction, ExpressionRewriteContext context) {
+        if (unboundFunction.getName().equalsIgnoreCase("element_at")) {
```

Comment:
it could not work correctly when user craate a UDF named element_at under database.